### ISSUE TITLE
Does the API still work

### ISSUE THREAD

**Issue by sjohnss** [hahnicity/flowzillow#2]
Thanks so much for this. Does the getResults API still work? I'm getting an empty JSON object regardless of the coordinates. Thanks!

**Comment by hahnicity** [comment]
Good question. I don't know. Let me get around to it tomorrow

**Comment by sjohnss** [comment]
Awesome. Thank you.

**Comment by hahnicity** [comment]
Seems to be working for me. Are you sure you're inputting the correct latitude and longitude?

It should be latlong1 = (lat upper left, long upper left) latlong2 = (lat lower right, long lower right)

**Comment by tonycpsu** [comment]
Coords are supposed to be in decimal degrees, right?

If so, I don't get anything very interesting coming back in the response:

``` python
>>> search_client.search((40.04838, -75.330033), (40.02091, -75.41595))
{u'map': {u'buildings': [], u'nearbyProperties': [], u'regions': [], u'nearbyBuildings': [], u'favorites': [], u'communities': [], u'mapResultsMode': u'MapConstants.MapResultsMode.Default', u'properties': []}, u'tempCustomRegionId': u'', u'mapStateMessage': u'', u'filters': {u'summary': u'<div id="sf11-title" class="title">Other Filters</div><div id="applied-summary">For Sale, Rent, MMM, Sold</div>'}, u'breadcrumb': {}}
```

Tried different search boxes at different zoom levels, etc.  My goal would be to get the zillow IDs for properties in the search rectangle.

**Comment by hahnicity** [comment]
Weird... I've been playing around with this for an hour and when I use your custom inputs in the Philadelphia area I can't get anything, but when I go to the networking view on my developer tools I'm able to find homes no problem when plugging in zillow generated coordinates.

I'm going to play around with this a bit more. One hypothesis I have is that zillow has some kind of guard up against coordinates they believe are artificially generated to stop software like this from getting their data. But I need more time to play around with this to map out the system more properly

**Comment by tonycpsu** [comment]
Hm, I tried it specifically with coordinates that I took out of the URLs of actual searches using the web interface and get the same results.

Could it be tracking user-agent / referer headers?

**Comment by hahnicity** [comment]
Weird. Maybe user-agent. Lemme get back to it when I have more time. Grad degrees are time sinks :(

**Comment by crichmond** [comment]
I tweaked the code in just a couple places to make it run with Python 3.5.1 and it runs butI have to say. No matter what coordinates I use I get empty results as well. Can't seem to resolve it.

**Comment by hahnicity** [comment]
Yeah. Did you try changing the user-agent headers to validate whether or not that was the problem?

**Comment by crichmond** [comment]
I thought about that, but I am not sure what user-agent header value to use.  I tried several chrome/windows user-agent strings with no avail. 
Yes. The answer is yes, I did.
